File: CloseWindow/CloseWindow.scala
import scala.util.control.Breaks
val b = new Breaks

println("窓は閉めましたよね? (y/n)")
b.breakable {
  val c = io.StdIn.readLine()
  if (!(c.equals("y"))) {
    println("窓を閉めてください。鳩が入ります")
  }
  else {
    b.break
  }
}
print("お疲れ様でした")

File: CloseWindow/CloseWindow.rs
use std::io;

fn main() {
    println!("窓は閉めましたよね？ (y/n)");
    loop {
        let mut c = String::new();
        io::stdin().read_line(&mut c).unwrap();
        if c != "y\n" {
            println!("窓を閉めて下さい。鳩が入ります。");
        } else {
            break;
        }
    }
    println!("お疲れ様でした。");
}
